sql/moz-fx-data-shared-prod/telemetry/releases_latest/view.sql

```diff
@@ -0,0 +1,70 @@
+WITH channels AS (
+  SELECT
+  "release" AS channel
+  UNION ALL SELECT "beta"
+),
+dates_and_channels AS (
+  SELECT
+    `date`, channel
+  FROM
+    UNNEST(
+      (SELECT GENERATE_DATE_ARRAY(DATE_SUB(CURRENT_DATE, INTERVAL 1 YEAR), CURRENT_DATE, INTERVAL 1 DAY) AS dates)
```

Comment:
This seems short-sighted, pun intended.  We have lots of queries that will want more than 1 year of release data, and there's just not that much of it -- I found ~10k rows with my queries.  Why not make it historical?